src/selectors.jl

```diff
@@ -1,122 +1,124 @@
-#https://developer.gnome.org/gtk2/stable/SelectorWidgets.html
+if libgtk_version >= v"3"     ### should work with v >= 2.4, but there is a bug for v < 3
```

Comment:
well, nominally FileChooserDialog was added to gtk in v2.4 according to their docs.  but i can't get it to work with the julia bindings using gtk2.  so i put a disclaimer there in case someone needed it badly enough that they wanted to fix it.